openpower-build-setup.sh

```diff
@@ -25,10 +25,21 @@ if [[ "${distro}" == fedora ]];then
   Dockerfile=$(cat << EOF
 FROM fedora:latest
 
-RUN dnf --refresh upgrade -y && \
-	dnf install -y vim gcc-c++ flex bison git ctags cscope expat-devel \
-	patch zlib-devel zlib-static perl
-RUN groupadd -g ${GROUPS} ${USER} && useradd -d ${HOME} -m -u ${UID} -g ${GROUPS} ${USER}
+RUN dnf --refresh upgrade -y && dnf install -y \
+	vim \
+	gcc-c++ \
+	flex \
+	bison \
+	git \
+	ctags \
+  	cscope \
+	expat-devel \
+	patch \
+	zlib-devel \
+	zlib-static perl
```

Comment:
Are these enough? I updated the readme in op-build recently and the list is a bit longer:
```
        sudo dnf install gcc-c++ flex bison git ctags cscope expat-devel patch \
          zlib-devel zlib-static texinfo perl-bignum "perl(XML::Simple)" \
          "perl(YAML)" "perl(XML::SAX)" "perl(Fatal)" "perl(Thread::Queue)" \
          "perl(Env)" "perl(XML::LibXML)" "perl(Digest::SHA1)" libxml2-devel \
          which wget unzip tar cpio python bzip2 bc findutils ncurses-devel
```